content/en/metrics/metrics-without-limits.md

```diff
@@ -22,17 +22,32 @@ algolia:
+Metrics without Limits™ provides you with the ability to configure tags on all metric types in-app. You can also customize aggregations on counts, rates, and gauges without having to re-deploy or change any code. With Metrics without Limits™, you can configure an allowlist of tags in-app to remain queryable throughout the Datadog platform. This automatically excludes nonessential tags attached to application-level or business metrics (for example, `host`). Additionally, you can configure a blocklist of tags to quickly drop and exclude tags that provide less business value to you. These functionalities are located in the [Metrics Summary][1] page.
 
 This page identifies key components of Metrics without Limits™ that can help you manage your custom metrics volumes within your observability budget.
 
-### Configuration of tags
+## Configuration of tags
 
-Click on any metric name to open its details sidepanel. Then click **Manage Tags** -> **"Include Tags..."** to configure the tags you'd like to remain as queryable on dashboards and monitors. The tag configuration modal pre-populates by default with an allowlist of tags that have been actively queried on dashboards, notebooks, monitors, and through API in the past 30 days (colored in blue with an icon). You can also include your own additional tags. Before selecting **Save**, an estimated new volume of indexed custom metrics that results from this potential tag configuration is displayed.
+### Allowlist of tags 
+1. Click any metric name to open its details side panel. 
+2. Click **Manage Tags** -> **"Include Tags..."** to configure the tags you'd like to remain as queryable on dashboards and monitors. 
+3. Review or add tags in the configuration modal. By default, the tag configuration modal pre-populates an allowlist of tags that were actively queried on dashboards, notebooks, monitors, and through the API in the past 30 days (represented by an icon). 
+4. Review the *Est. New Volume* of indexed custom metrics that results from this potential tag configuration.
+4. Click **Save**.
 
-{{< img src="metrics/mwl_tag_config.mp4" alt="Configuration of Tags" video=true >}}
+{{< img src="metrics/mwl_example_include_tags.mp4" alt="Configuration of Tags with Allowlist" video=true >}}
 
 
+### Blocklist of tags 
+1. Click on any metric name to open its details side panel.
+2. Click **Manage Tags** -> **"Exclude Tags…"** to drop tags you don't want to query. 
+3. Review or add tags in the configuration modal. These tags are **not** queryable on dashboards and monitors.
+4. Hover over a tag to see if it has been actively queried on dashboards, notebooks, monitors, and through the API in the past 30 days. 
```

Comment:
Merge steps 3/4 so we have equal number of steps for inclusion/exclusion

Reply:
I think it's ok to have more steps for one over the other. Having bigger blocks of information can be harder to digest.

Reply:
Since steps 3 and 4 talk about looking for queryable tags in the configuration modal, it would make sense to combine the two together